### ISSUE TITLE
warm up

### ISSUE THREAD

**Issue by bcc2974874275** [happyharrycn/actionformer_release#108]
May I ask why warmup only increased the number of epochs trained, rather than increasing it with a small learning rate (similar to MASK RCNN)?

**Comment by tzzcl** [comment]
We did the linear warmup learning rate schedule in warmup epochs, please see [LinearWarmupScheduler](https://github.com/happyharrycn/actionformer_release/blob/main/libs/utils/train_utils.py#L144) for details.

**Comment by happyharrycn** [comment]
Closed due to inactivity.
